clang/test/Driver/sycl-offload-new-driver.cpp

```diff
@@ -141,27 +141,14 @@
 // MULT_TARG_PHASES: 15: backend, {14}, assembler, (host-sycl)
 // MULT_TARG_PHASES: 16: assembler, {15}, object, (host-sycl)
 
-/// Test option passing behavior for clang-offload-wrapper options.
-// RUN: %clangxx --target=x86_64-unknown-linux-gnu -fsycl --offload-new-driver \
-// RUN:          -Xsycl-target-backend -backend-opt -### %s 2>&1 \
```

Comment:
Per my understanding, we still need to pass `-backend-opt` to clang-linker-wrapper through `device-compiler` option.
so this test should be just updated to use proper clang-linker-wrapper option.
Similar for linker options.